documentation/TechnicalDesignDocument.tex

```diff
@@ -96,6 +96,26 @@
     In this section build automation and its constituent components will be discussed.
     How we applied it, the software used and the targets and platforms supported are among a few.
 
+    %------------------------------------------------------------------------------------------------------------------------------------------------------- Continuing Development
+    \section{Continuing Development}
+    This section describes the plan on how to continue with development in the best way.
+    The team worked on quite some user stories from april 9th, to june 15th.
+    The following functionalities are most critical at the moment to start working on:
+    \myparagraph{Instantiating maramafications from a .mar file.}
+    A beginning is made at implementing the .mar file.
+    The .mar file is basically a json file with some agreements on structure.
+    The goal of the .mar file is, whenever the file is loaded in, a maramafication is instantiated.
```

Comment:
Not sure if this is clear. But as far as I know it just means it is added to the list of all possible Ms (think the sidebar in the view project).

Reply:
Added a clarification.

Reply:
The goal of the .mar file is to instantiate a maramafication when the file is loaded in.

Maybe?